x/evm/client/cli/tx.go

```diff
@@ -333,8 +328,36 @@ func CmdDeployErc20() *cobra.Command {
 			privHex := hex.EncodeToString(priv.Bytes())
 			key, _ := crypto.HexToECDSA(privHex)
 
+			rpc, err := cmd.Flags().GetString(FlagRPC)
+			if err != nil {
+				return err
+			}
+			nonceQuery := fmt.Sprintf("{\"jsonrpc\": \"2.0\",\"method\": \"eth_getTransactionCount\",\"params\":[\"%s\",\"latest\"],\"id\":\"send-cli\"}", crypto.PubkeyToAddress(key.PublicKey).Hex())
```

Comment:
we may want to use pending here, if we run it multiple times via scripting

Reply:
updated